notebooks/Notebook2.md

```diff
@@ -17,18 +12,34 @@ exciting milestones, preliminary results, etc. Feel free to include images
 (e.g., a sketch of the design or a screenshot of a running program), links to
 code, and any other resources that you think will help clearly convey your
 design process.
+-- REMOVE 
+Some reps to look at https://github.com/hmc-cs111-spring2023/hw-7-piconot-rydermitchell-hw7/blob/main/build.md
+https://github.com/hmc-cs111-spring2023/hw-7-sample-solution/tree/main/src/main/scala/piconot/external
+
+--
+
+This week I did not work as much as I would have liked. I mainly just took our sample solution from hw7 and converted it into my artifact repo. I want to use its external solution as structure for my own.  The main progress I made was on a language for parseing my language. Check it out in artifact [design.md](https://github.com/hmc-cs111-spring2023/artifact-AdamrBeckwith/blob/main/design.md) would love feedback on that setup. (I am considering the writing there as part of this notebook) 
```

Comment:
The syntax looks great so far- I'd definitely advise checking out some  combinators so that you can write things like 1d6 (instead of 1 d 6). There's also definitely a complexity (which we discussed a bit in class) about things like 1d6 + 1. Since 1d6 is a function, 1d6 + 1 should probably also be a function, which means that you need to have some way to arithmetically combine numbers with functions (and functions with functions, e.g. 1d8 + 1d6). I'm also still in favor of changing the name stat, since it seems a bit confusing in relation to TTRPG stats,  but I'm not totally sure what to replace it with- maybe prob?

Reply:
I think you also might want to carve out some more space in your syntax for some customizability- e.g. non-standard dice, or dice that have some sort of effect like exploding 1s/6s.